### DISCUSSION TITLE
Adding WSL support for Windows

### DISCUSSION THREAD

**Discussion by rainwashed** [GitSquared/edex-ui#1140] [Ideas]
I think EDEX would be a great alternative compared to the Windows Terminal for WSL (Windows Subsystem for Linux). I do not know how difficult this is as I am not a skilled programmer but I think looking into the Windows Terminal [code](https://github.com/microsoft/terminal) and seeing how it works could help. I am not too sure though but it would be cool and somewhat beneficial to people who use WSL and still want this sexy terminal.

**Comment by Animus-Surge** [comment]
You can set the terminal option in the settings to `wsl` and it works flawlessly

**Reply by rainwashed** [reply]
Oh wow, never knew that! I will definitely look at that.

**Reply by GitSquared** [reply]
You should take a look at the wiki: https://github.com/GitSquared/edex-ui/wiki and the various customization options documented there ;)
